React InstantSearch Hooks/query-suggestions/src/QuerySuggestions.tsx

```diff
@@ -0,0 +1,56 @@
+import { createQuerySuggestionsPlugin } from '@algolia/autocomplete-plugin-query-suggestions';
+
+import {
+  useInstantSearch,
+  useMenu,
+  useSearchBox,
+} from 'react-instantsearch-hooks-web';
+
+import { Autocomplete } from './Autocomplete';
+import { searchClient } from './searchClient';
+
+export function QuerySuggestions() {
```

Comment:
The name `QuerySuggestions` doesn't imply that it mounts a search box.

Alternative suggestions:

- AutocompleteSearchBox
- SearchBoxWithSuggestions

Reply:
I went with `SearchBoxWithSuggestions`, thanks for the suggestions!